Text-Tools.md

```diff
@@ -113,30 +113,12 @@
 ## ▷ Grammar Check
 
 * ⭐ **[Scribens](https://www.scribens.com/)**
-* ⭐ **[Jenni](https://jenni.ai/)**
-* [Wordtune](https://www.wordtune.com/)
-* [HemingwayApp](https://hemingwayapp.com/)
-* [Grammark](https://grammark.org/)
-* [SlickWrite](https://www.slickwrite.com/)
-* [Ginger Software](https://www.gingersoftware.com/grammarcheck)
 * [LanguageTool](https://languagetool.org/)
-* [Grammarly](https://www.grammarly.com/)
-* [Grammica](https://grammica.com/)
-* [Ludwig](https://ludwig.guru/)
-* [Scribbr](https://www.scribbr.com/)
-* [Writer](https://writer.com/)
-* [WordCounter](https://wordcounter.net/)
-* [RewriteTools](https://www.rewritertools.com/)
-* [TextGears](https://textgears.com/check-grammar-online)
-* [PolishMyWriting](https://www.polishmywriting.com/)
-* [Reverso Spellcheck](https://www.reverso.net/spell-checker/english-spelling-grammar/)
-* [spellcheckplus](https://spellcheckplus.com/)
-* [GrammarCheck](https://www.grammarcheck.net/)
-* [ContentRow](https://www.contentrow.com/)
-* [BibMe](https://www.bibme.org/)
-* [DeepL Write](https://www.deepl.com/write)
-* [PaperPal](https://paperpal.com/)
-* [Expresso](https://www.expresso-app.org/)
+* [Grammarly](https://www.grammarly.com/grammar-check)
+* [Scribbr](https://www.scribbr.com/grammar-checker/)

```

Comment:
Replace with https://quillbot.com/grammar-check since it just uses that
![image](https://github.com/fmhy/FMHYedit/assets/98007575/f23b17df-f244-42ef-9bf0-48b1023811ff)